apps/okreads-e2e/src/integration/reading-list.spec.ts

```diff
@@ -30,31 +30,9 @@ describe('When: I use the reading list feature', () => {
       'My Reading List'
     );
     
-    cy.get('[data-testing="toggle-finish-book-cta"].mat-icon-button-gray').first().click();
+    cy.get('[data-testing="finish-book-cta"]:enabled').first().click();
 
-    cy.get('[data-testing="toggle-finish-date"]').first().should('contain.text', 'Finish reading');
+    cy.get('[data-testing="toggle-finish-date"]').first().should('contain.text', 'Finished reading');
```

Comment:
Please change data-testing value to finish-date.

Reply:
Done